src/twisted/protocols/ftp.py

```diff
@@ -736,10 +739,20 @@ class FTP(basic.LineReceiver, policies.TimeoutMixin, object):
     passivePortRange = xrange(0, 1)
 
     listenFactory = reactor.listenTCP
+    _encoding = 'latin-1'
 
     def reply(self, key, *args):
         msg = RESPONSE[key] % args
-        self.sendLine(msg)
+        self.sendEncodedLine(msg)
+
+
+    def sendEncodedLine(self, line):
+        """
+        (Private) Encodes and sends a line
```

Comment:
I'm more than happy to change the name, I was following what the rest of the code in this file did.